### PR TITLE
fix some deploy problems

### PR DESCRIPTION
**Pull request by skeeey**

### CODE REVIEW

File: deploy/nucleus-hub/operator.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ kind: Deployment
 apiVersion: apps/v1
 metadata:
   name: nucleus-hub
-  namespace: open-cluster-management-core
+  namespace: open-cluster-management
```
Comment (deads2k) [inline deploy/nucleus-hub/operator.yaml:5]: I think we just tweaked this open-cluster-management-hub in the api repo here https://github.com/open-cluster-management/api/pull/25

Comment (skeeey) [reply deploy/nucleus-hub/operator.yaml:5]: This namespace is for nucleus operator, it is not for hub core components, the nucleus operator and hub core have a different namespace, the  hub core namespace is open-cluster-management-hub

**Pull request merged**